workflows.md

```diff
@@ -0,0 +1,41 @@
+# WORKFLOWS
+
+This document outlines some typical workflows of using COMPAS IFC.
+
+# Stand-alone
+
+Using COMPAS IFC directly in python scripts to build IFC hierarchies, assign geometry representations, define properties, and export to IFC files.
+
+### TODO
+- [ ] Define default behavior to store full COMPAS geometry metadata to IFC properties, such as Mesh Vertex attributes.
+
+# From COMPAS Model
+
+Craeting a `COMPAS Model` first. And use a built-in `to_ifc_model()` funtion to directly convert to an COMPAS IFC model, or use `to_ifc(path)` to directly export to an IFC file.
+
+### TODO
+- [ ] Extend the `COMPAS Model` class to `BimModel`. Perhaps in a new package `COMPAS BIM`?
```

Comment:
perhaps `BIModel` as in `Building Information Model`?
i would not create an additional package, but rather have multiple types of models live in `compas_model`. at least for now...